README.md

```diff
@@ -91,6 +91,7 @@
 - [useweb3.xyz/tutorials](https://www.useweb3.xyz/tutorials) - A curated list of free, community tutorials that are based around specific projects, tasks or challenges.
 - [WTF Solidity](https://github.com/AmazingAng/WTF-Solidity) - An open-source and community reviewed tutorial in Chinese and English, covering intro, advanced, and application topics.
 - [WTF Ethers](https://github.com/WTFAcademy/WTF-Ethers) - An open-source and community reviewed Ethers.js tutorial in Chinese covering intro and advanced topics [Chinese Language - 中文版].
+- [Alchemy University](https://university.alchemy.com/) - Earn your web3 degree. Fast track your web3 career through courses, projects and code. Totally free.
```

Comment:
Please provide a description of the actual solidity-related content as requested in the guidelines. This is a marketing tag line